Podcast/BookmarkTableViewCell.swift

```diff
@@ -171,7 +171,7 @@ class BookmarkTableViewCell: UITableViewCell {
     }
     
     func setPlayButtonToState(isPlaying: Bool) {
-        playButton.isEnabled = !isPlaying
+        playButton.isSelected = isPlaying
```

Comment:
I don't think we want to use `isEnabled` as that determines if you can click on a UIButton. I see what you're talking about with that bug, though, and I just realized that I forgot to change `isEnabled` to `isSelected` in a couple places. I'm about to commit a change that fixes it for all views. I think some people were using `isEnabled` and others were using `isSelected` which is what was causing the bug.